src/main/kotlin/com/cognifide/gradle/aem/instance/action/AwaitAction.kt

```diff
@@ -84,6 +86,42 @@ open class AwaitAction(project: Project, val instances: List<Instance>) : Abstra
         progressLogger.completed()
     }
 
+    private fun prepareSynchronizers(): List<InstanceSync> {
+        return instances.map { instance ->
+            val init = instance is LocalInstance && !LocalHandle(project, instance).initialized
+
+            InstanceSync(project, instance).apply {
+                val sync = this
+
+                requestConfigurer = { request ->
+                    request.config = RequestConfig.custom()
+                            .setConnectTimeout(timeout)
+                            .setSocketTimeout(timeout)
+                            .build()
+                }
+
+                if (init) {
```

Comment:
when `-Dadmin.password=xxx` is specified, aem initially is accessible still by default credentials, but when about 40% of bundles are active then that password is being set and await is detecting unauthorized and then starting from that moment is using target credentials